### ISSUE TITLE
Add Slow Bookkeeper Servers to Placement Policy for read ordering

### ISSUE THREAD

**Issue by philipsu522** [apache/bookkeeper#709]
Feature Request

1. Please describe the feature you are requesting.
The bookkeeper client currently only differentiates between bookkeeper hosts that are available, readonly or unavailable when reading. However, clients will continue encountering lag if a single bookkeeper host in the quorum is slow in serving requests.

The reason is that the bookkeeper client picks bookkeeper hosts to read from in a round robin fashion. Therefore, if the read quorum is 3 and one of the 3 hosts are slow, then every third read will be served by a slow host, which results in lag building up.

The proposed solution for this is to maintain a list of slow bookkeeper hosts, which are tried only after readonly bookkeeper hosts are tried. Bookkeeper hosts can be categorized as "slow" when a speculative timeout occurs on that bookkeeper host.

2. Indicate the importance of this issue to you (blocker, must-have, should-have, nice-to-have). Are you currently using any workarounds to address this issue?
should-have
